### PR TITLE
Issue698 heat star comb

### LINKED ISSUE #698: Error: AixLib.Utilities.Interfaces.HeatStarComb has the same name as its type AixLib.Utilities.Interfaces.Star
Hi!
I'm building a model to simulate thermal loads in a room using AixLib components on OpenModelica. However, when I check the project the following warning message arises:

Component Star in env: AixLib.Utilities.Interfaces.HeatStarComb has the same name as its type AixLib.Utilities.Interfaces.Star.
        This is forbidden by Modelica specification and may lead to lookup errors.

Tried to check some of AixLib's examples such as OFD_1Jan ans OFDHeatLoad and I get the same warning message. For all these cases, an error occurs while flattening the model which I suspect to be related with the former warning.

### PR DESCRIPTION
**Pull request by PMehrfeld**
This closes #698 

Furthermore, in the last commit I deleted a self connection of therm_outside port in the wall model.

### CODE REVIEW

File: AixLib/Utilities/Interfaces/Adaptors/HeatStarToComb.mo [hunk 1]
```diff
@@ -1,23 +1,21 @@
...
- <li><i>April 01, 2014 </i> by Moritz Lauster:<br/>Renamed</li>
- <li><i>April 10, 2013&nbsp;</i> by Ole Odendahl:<br/>Formatted documentation appropriately </li>
- <li>by Mark Wesseling:<br/>Implemented.</li>
- </ul>
- </html>"));
-end HeatStarToComb;
+within AixLib.Utilities.Interfaces.Adaptors;
+model HeatStarToComb
+  AixLib.Utilities.Interfaces.HeatStarComb thermStarComb annotation(Placement(transformation(extent = {{-120, -10}, {-76, 36}}), iconTransformation(extent = {{-116, -24}, {-72, 22}})));
+  AixLib.Utilities.Interfaces.Star star annotation(Placement(transformation(extent = {{84, 38}, {124, 78}}), iconTransformation(extent = {{84, 38}, {124, 78}})));
```
Comment (AnaConstantin) [inline AixLib/Utilities/Interfaces/Adaptors/HeatStarToComb.mo]: Shouldn't we change this to rad (from star) as well? If so, also in the convert script. 
If I understand this correctly, the issue should have come up here as well. Did OpenModelica not complain?
Actually I think we should proceed another way: HeatPort_a is a good name, Star isn't. I suggest RadPort, starting with AixLib.Utilities.Interfaces.Star to AixLib.Utilities.Interfaces.RadPort.

File: AixLib/ThermalZones/HighOrder/Components/Walls/Wall.mo [hunk 2]
```diff
@@ -1,202 +1,201 @@
...
-  // **** connections for outside wall with window and sunblind****
-  //******************************************************************
-  if outside and withWindow and withSunblind then
-    connect(Sunblind.Rad_Out[1], windowSimple.solarRad_in) annotation(Line(points={{-22.15,
-            -6.7},{-18,-6.7},{-18,-27.2},{-13.7,-27.2}},                                                                                         color = {255, 128, 0}));
-    connect(Sunblind.Rad_In[1], SolarRadiationPort) annotation(Line(points={{-42.85,
-            -6.7},{-81,-6.7},{-81,89},{-106,89}},                                                                                  color = {255, 128, 0}));
-  end if;
-  connect(heatStarToComb.thermStarComb, thermStarComb_inside) annotation(Line(points = {{78.4, -1.1}, {78.4, -1.05}, {102, -1.05}, {102, 0}}, color = {191, 0, 0}));
-  connect(port_outside, port_outside) annotation(Line(points = {{-98, 4}, {-98, 4}}, color = {191, 0, 0}, pattern = LinePattern.Solid));
```
Comment (AnaConstantin) [inline AixLib/ThermalZones/HighOrder/Components/Walls/Wall.mo]: This is what you deleted, yes? Very good.

File: AixLib/Utilities/Interfaces/Adaptors/HeatStarToComb.mo (hunk 1 above)
Comment (PMehrfeld) [reply AixLib/Utilities/Interfaces/Adaptors/HeatStarToComb.mo]: @AnaConstantin : I am pretty sure that it is case sensitive. Thus, OpenModelica did not complain.
You are absolutely right with your suggestions!

I will commit new proposals

File: AixLib/ThermalZones/HighOrder/Components/Walls/Wall.mo (hunk 2 above)
Comment (PMehrfeld) [reply AixLib/ThermalZones/HighOrder/Components/Walls/Wall.mo]: Yes, exactly. Sorry, due to ASCII <-> UTF8 encoding issues GitHub marks the whole file as different.
Although, one can hide whitespace changes in the diff settings in the upper right corner in GitHub's diff viewer.

Comment (AnaConstantin) [reply AixLib/ThermalZones/HighOrder/Components/Walls/Wall.mo]: It worked. Thank you!

**Comment by PMehrfeld** [comment]
@AnaConstantin : Yes, still working on the encoding issue. The problem with renaming comes along with lots of file changes and also encoding changes. Maybe we solve #587 first.
However, I will update this branch es soon as possible and let you know, when I am done.

- [ ] Conversion script must cover all changes due to refactoring.

**Comment by PMehrfeld** [comment]
@AnaConstantin: All changes are done. 
If everything is okay, I am looking forward to your approval ;)

**Pull request merged**